ios/browser/api/brave_domains/service_domains.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_BRAVE_DOMAINS_SERVICE_DOMAINS_H_
+#define BRAVE_IOS_BROWSER_API_BRAVE_DOMAINS_SERVICE_DOMAINS_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSInteger BraveServicesEnvironmentIOS
+    NS_TYPED_ENUM NS_SWIFT_NAME(BraveDomains.ServicesEnvironment);
+
+OBJC_EXPORT BraveServicesEnvironmentIOS const
+    BraveServicesEnvironmentIOSDevelopment;
+OBJC_EXPORT BraveServicesEnvironmentIOS const
+    BraveServicesEnvironmentIOSStaging;
+OBJC_EXPORT BraveServicesEnvironmentIOS const
+    BraveServicesEnvironmentIOSProduction;
+
+OBJC_EXPORT
+@interface BraveDomains : NSObject
+@property(class, nonatomic, readonly) BraveServicesEnvironmentIOS environment;
+
++ (BraveServicesEnvironmentIOS)environmentForPrefix:(NSString* _Nonnull)prefix
```

Comment:
This isn't used anywhere?

Reply:
Yeah. I haven't use it for now but I was thinking that if I add specific switches for leo and vpn overrides, I'd definitely need it. So I exported it now just in case.